bounswe/bounswe2017group5
# Webfrontend with join/leave group and news feed without news

## About

I need this merge to proceed with the implementation of templates. I need the latest updates on the master branch, from which I will be branching off again.

### Changes

- Join/leave group functionality added.
- News feed page has been created and ready to get functional.

## Testing

How can this contribution be tested? Use the following structure:

- Open a group page and try to join it via the "Join Group" button.
- Note that the group member count updates only after a page refresh...
- Leave the group page and then re-enter, you shall see that you are still in the group.
- Try similar things with the "Leave Group".
